tests/fv/__init__.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sh
+from sh import docker
+
+
+def setup_package():
+    pwd = sh.pwd().stdout.rstrip()
+
+    docker_exec = docker.bake("exec")
+    host1_exec = docker_exec.bake("-t", "host1", "bash", "-c")
```

Comment:
Not sure I like this.

Better to query which docker in docker hosts there actually are, and then remove the inner containers for all of them.  Perhaps naming them something more uniquely identifiable will make this a safer operation, or perhaps querying the image type.